apps/prairielearn/src/tests/questionSharing.test.ts

```diff
@@ -47,26 +48,85 @@ async function setSharingName(courseId, name) {
   });
 }
 
-async function accessSharedQuestionAssessment() {
-  const assessmentsUrl = `${baseUrl}/course_instance/${testCourseId}/instructor/instance_admin/assessments`;
+async function accessSharedQuestionAssessment(course_id: string) {
+  const assessmentsUrl = `${baseUrl}/course_instance/${course_id}/instructor/instance_admin/assessments`;
   const assessmentsPage = await fetchCheerio(assessmentsUrl);
-  const sharedQuestionAssessmentUrl =
-    siteUrl +
-    assessmentsPage.$(`a:contains("Test of Consuming Questions From Another Course")`).attr('href');
+  const assessmentLink = assessmentsPage.$(`a:contains("Test assessment")`);
+  assert.lengthOf(assessmentLink, 1);
+  const sharedQuestionAssessmentUrl = siteUrl + assessmentLink.attr('href');
   const res = await fetchCheerio(sharedQuestionAssessmentUrl);
   assert.equal(res.ok, true);
   return res;
 }
 
 describe('Question Sharing', function () {
   this.timeout(80000);
-  before('set up testing server', helperServer.before(EXAMPLE_COURSE_PATH));
+  before('set up testing server', helperServer.before());
   after('shut down testing server', helperServer.after);
 
-  before('ensure course has question sharing enabled', async () => {
+  before('ensure question sharing is globally enabled', async () => {
     await features.enable('question-sharing');
   });
 
+  // Rather than using the example course and test course, we'll set up two courses
+  // from scratch. This is necessary because the example course has hardcoded behavior
+  // to prevent all question sharing features from working.
+  let sharingCourse: Course;
+  let consumingCourse: Course;
+  before('construct and sync course', async () => {
+    const sharingCourseData = syncUtil.getCourseData();
+    sharingCourseData.course.name = 'SHARING 101';
+    sharingCourseData.questions = {
+      [SHARING_QUESTION_QID]: {
+        uuid: '00000000-0000-0000-0000-000000000000',
```

Comment:
why not just generate a random UUID here and below? That seems more future-proof to me.

Reply:
🤷 I don't think it matters much either way?